### PR TITLE
Fix: top-padding in tag-line

### PR DESCRIPTION
**Pull request by rajatgupta24**
# Description
Fixing the overlapping of heading in mobile view.

Fixes # (issue) 
#3522

# Checklist:

- [x] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [x] Any dependent changes have already been merged.

### CODE REVIEW

File: layouts/partials/navbar.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
 <nav class="js-navbar-scroll navbar fixed-top navbar-expand-lg navbar-light bg-light {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
 	<a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
 		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span>
-	</a><br>
+	</a>
+	<!-- <br> -->
```
Comment (ankitm123) [inline layouts/partials/navbar.html:10]: if it's commented, just remove it.

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,1843 @@
 {
   "name": "jx-docs",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (ankitm123) [inline package-lock.json:4]: why this change?

File: layouts/partials/navbar.html (hunk 1 above)
Comment (rajatgupta24) [reply layouts/partials/navbar.html:10]: Ok

File: package-lock.json (hunk 2 above)
Comment (rajatgupta24) [reply package-lock.json:4]: Didn't made any change in package-lock.json, just run `npm i` & it made a lot of changes
